build-tools/scripts/jenkinsfile

```diff
@@ -0,0 +1,254 @@
+                        ]],
+                        group: 'Tests times', logarithmic: true, style: 'line', yaxis: 'ms'
+                    )
+                    plot(
+                        title: 'Locale',
+                        csvFileName: 'plot-locale-times.csv',
+                        csvSeries: [[
+                            displayTableFlag: true, file: 'TestResult-Xamarin.Android.Locale_Tests-times.csv', inclusionFlag: 'OFF'
+                        ]],
+                        group: 'Tests times', logarithmic: true, style: 'line', yaxis: 'ms'
+                    )
+                    plot(
+                        title: 'Runtime test sizes',
+                        csvFileName: 'plot-runtime-test-sizes.csv',
+                        csvSeries: [[
+                            displayTableFlag: true, file: 'TestResult-Mono.Android_Tests-values.csv', inclusionFlag: 'OFF'
+                        ]],
+                        group: 'Tests size', logarithmic: true, style: 'line', yaxis: 'ms'
+                    )
+                    plot(
+                        title: 'Runtime merged',
+                        csvFileName: 'plot-runtime-merged-test-times.csv',
+                        csvSeries: [[
+                            displayTableFlag: true, file: 'TestResult-Mono.Android_Tests-times.csv', inclusionFlag: 'OFF'
+                        ]],
+                        group: 'Tests times', logarithmic: true, style: 'line', yaxis: 'ms'
+                    )
+                    plot(
+                        title: 'Xamarin.Forms app startup',
+                        csvFileName: 'plot-xamarin-forms-startup-test-times.csv',
+                        csvSeries: [[
+                            displayTableFlag: true, file: 'TestResult-Xamarin.Forms_Test-times.csv', inclusionFlag: 'OFF'
+                        ]],
+                        group: 'Tests times', logarithmic: true, style: 'line', yaxis: 'ms'
+                    )
+                    plot(
+                        title: 'Xamarin.Forms app',
+                        csvFileName: 'plot-xamarin-forms-tests-size.csv',
+                        csvSeries: [[
+                            displayTableFlag: true, file: 'TestResult-Xamarin.Forms_Tests-values.csv', inclusionFlag: 'OFF'
+                        ]],
+                        group: 'Tests size', logarithmic: true, style: 'line', yaxis: 'ms'
+                    )
+
+                    // UNDONE: Safe to remove?
+                    //plot(
+                    //    title: '[History] (before the graphs were split up)',
+                    //    csvFileName: 'plot-build-times.csv',
+                    //    csvSeries: [[
+                    //        displayTableFlag: true, file: 'TestResult-MSBuild-times.csv', inclusionFlag: 'OFF'
+                    //    ]],
+                    //    group: 'Build times', logarithmic: true, style: 'line', yaxis: 'ms'
```

Comment:
When we switch to a pipeline build will we lose plot history anyway?

I found I had to leave this in the config to keep the old graph data around.

Reply:
Yes, per our earlier conversation since we can't simply convert the existing XA configuration to a pipeline build and so any history won't carry forward.  The plan is to rename the pipeline definition to xamarin-android once we validate that it produces the expected and correct outputs (including plots and test results).  Also rename the existing XA freestyle definition to something like 'xamarin-android-deprecated' (btw: let me know if there's a name you prefer).  Plot history for previous builds will remain available via that deprecated definition.